src/js/tabs/osd.js

```diff
@@ -565,7 +580,13 @@ OSD.loadDisplayFields = function() {
             defaultPosition: 1,
             draw_order: 120,
             positionable: true,
-            preview: 'R:2:200:P',
+            variants: [
+                'osdTextElementVTXchannelVariantFull',
+                'osdTextElementVTXchannelVariantPower',
+                ],
```

Comment:
Please fix indentation.
```suggestion
           ],
```